docs/docs/guides/labs/dg/dagster-definitions.md

```diff
@@ -1,37 +1,35 @@
 ---
 title: 'Scaffolding traditional Dagster definitions'
-sidebar_position: 500
+sidebar_position: 200
 ---
 
 import Preview from '../../../partials/\_Preview.md';
 
 <Preview />
 
-`dg` can be used to help scaffold traditional Dagster definitions such as schedules, sensors, and assets. When using a project that has been scaffolded using `dg`, any new definitions added underneath the `defs` directory will be automatically loaded into the top-level `Definitions` object. This allows new definitions to be easily added to a project without needing to explicitly import these definitions to your top-level definitions file.
+`dg` can be used to help scaffold traditional Dagster definitions such as [assets](/guides/build/assets/), [schedules](/guides/automate/schedules/), and [sensors](/guides/automate/sensors/). When you use a project that has been scaffolded using `dg`, any new definitions added underneath the `defs` directory will be automatically loaded into the top-level `Definitions` object. This allows you to easily add new definitions to your project without needing to explicitly import these definitions to your top-level definitions file.
```

Comment:
Part of me thinks we should remove the term traditional, since this will be "the way" to do things. While possibly calling out that it can be used to also scaffold components?

```suggestion
`dg` can be used to help scaffold Dagster definitions such as [assets](/guides/build/assets/), [schedules](/guides/automate/schedules/), and [sensors](/guides/automate/sensors/). When you use a project that has been scaffolded using `dg`, any new definitions added underneath the `defs` directory will be automatically loaded into the top-level `Definitions` object. This allows you to easily add new definitions to your project without needing to explicitly import these definitions to your top-level definitions file.
```